src/app/globals.css

```diff
@@ -106,3 +106,10 @@
     transform: translateX(0);
   }
 }
+
+h2 + .InputAdjacentText:focus {
+  top: -0.75rem;
+  font-size: 14px;
+  padding: 0 0.5rem;
+  color: hsl(217, 91%, 60%); /*blue-500*/
+}
```

Comment:
could you try do this with tailwind, I found a really good video which showcases pretty much exactly how to do this

https://www.youtube.com/watch?v=nJzKi6oIvBA